walkthroughs/howto-k8s-http2/deploy.sh

```diff
@@ -12,6 +12,11 @@ if [ -z $AWS_DEFAULT_REGION ]; then
     exit 1
 fi
 
+if [ -z $VPC_ID ]; then
```

Comment:
This being checked further down in the [file](https://github.com/aws/aws-app-mesh-examples/blob/master/walkthroughs/howto-k8s-http2/deploy.sh#L96)

Reply:
Yes but wouldn't it be better to check beforehand if the variable exist before trying to deploy cloudmap?

Reply:
Well, `VPC_ID` is only required if the cloudmap namespace doesn't exist. It probably wouldn't make sense to make it required even when that is not required.